src/material/form-field/form-field.scss

```diff
@@ -165,3 +165,11 @@
 .mdc-notched-outline--upgraded .mdc-floating-label--float-above {
   max-width: calc(100% * 4 / 3 + 1px);
 }
+
+.mat-mdc-form-field {
+  &.mat-form-field-appearance-fill {
+    .mat-mdc-floating-label {
```

Comment:
Nit: The selector could be condensed into a single line.

Reply:
Hi @devversion 

I have made required changes.